api/v1beta1/foundationdbrestore_types.go

```diff
@@ -50,8 +50,15 @@ type FoundationDBRestoreList struct {
 type FoundationDBRestoreSpec struct {
 	// DestinationClusterName provides the name of the cluster that the data is
 	// being restored into.
+	// Deprecated: Use ClusterName instead.
```

Comment:
`DestinationClusterName` matches the naming in the `fdbrestore` command. Even though we don't support it yes in the operator, there are some cases where you want to pass a source cluster to a restore, so that you can pull timekeeper data from the original cluster. This would make `ClusterName` ambiguous.

Reply:
That makes sense 👍